manticore/platforms/evm.py

```diff
@@ -738,6 +738,14 @@ def extend_with_zeroes(b):
         self._calldata_size = len(self.data)
         self._valid_jmpdests = set()
 
+    @property
+    def pc(self):
+        return self._pc
+
+    @pc.setter
+    def pc(self, pc):
+        self._pc = simplify(pc)
```

Comment:
This is weird as this PR does not include any additional uses of `self.pc` or `self._pc`.

We initialise `self.pc` in here:
https://github.com/trailofbits/manticore/blob/master/manticore/platforms/evm.py#L722-L723

And before this PR the only use of `self._pc` in this file is in:
* https://github.com/trailofbits/manticore/blob/master/manticore/platforms/evm.py#L1125

Either I am missing something or the latter is probably a bug and maybe should be a `self.pc = ...`, and the `self.pc = 0` should be changed to `self._pc = 0`?

Also: are we sure this does not break anything? It passes the tests but do we have tests for everything, like tainting?

@feliam you might have some comment here too

Reply:
> this PR does not include any additional uses of self.pc or self._pc.
Right.  It is meant to influence the existing writes to `self.pc`.

> ...the latter is probably a bug...
I looked at this and thought it should be okay following this PR. But I agree with your sentiment that it should be `self.pc = ...`. I will change it.

> ...the self.pc = 0 should be changed to self._pc = 0?
You're right. I'll make that change as well.

Reply:
https://github.com/trailofbits/manticore/blob/19ba4b6339c350368cce904573caefb689a049f2/manticore/platforms/evm.py#L1126 is probably a big _without_ this PR